content/community/roadmap.md

```diff
@@ -1,21 +0,0 @@
----
-title: "Roadmap"
-date: 2018-07-16T14:49:14-07:00
-draft: false
-weight: 100
-aliases: [/roadmap]
----
-Read OpenCensus’s journey ahead: [platforms and languages](https://opensource.googleblog.com/2018/05/opencensus-journey-ahead-part-1.html).
-
-
-#### Languages
-
-{{<languages>}}
-
-#### Exporters
```

Comment:
Ditto, why this removal?

Reply:
Same as above, this should either have its own page on "supported backends" or should shift to the languages page

Reply:
Re-added the file